### DISCUSSION TITLE
有没有 win11 用户有低概率碰到计算机卡屏问题（画面完全卡死只能重启）

### DISCUSSION THREAD

**Discussion by szh-bash** [nextai-translator/nextai-translator#454] [Q&A] [answered]
最近我的计算机开始出现卡死现象（画面停止音频停止键鼠无响应只能重启恢复），1~4天一次的频率（共计6次），目前在找 Bug 原因。
因为每次卡屏貌似都开着 chrome 放视频，所以我感觉 chrome 可能是一个“强相关”因素，然后倒推了下时间发现第一次出现卡屏问题 (3.10) 是开始用此插件（导入至 chrome）后的几天（貌似是 3.6 ），3.8 安装 web stroe 版插件但没启用。
不知道还有没有其他朋友遇到过这问题。
Chrome: 111.0.5563.110
OS: win11-22H2-Pro-x64
CPU: i7-7700hq
SSD: 990 Pro 1T
![image](https://user-images.githubusercontent.com/27895812/227220984-5dc68dec-d24b-48bc-b963-f8600c9d32df.png)

**Comment by szh-bash** [comment]
![image](https://user-images.githubusercontent.com/27895812/227262303-b891040b-58cd-42f9-8c0c-5bfd05c64bb7.png)

**Reply by AnYiEE** [reply]
应该是系统本身有问题，浏览器中的js并没有直接访问系统的能力，你完全可以排除卡住是因为浏览器的xx插件所导致的。

我也遇到过类似的现象。我用笔记本，那段时间外接了一台显示器，设置了屏幕超时。当系统自动关闭屏幕后，我再尝试唤醒就会时不时那样，而不外接显示器只用内置显示器且有屏幕超时的时候从没出现过这个现象。索性我就关了屏幕超时，让屏幕常亮了，不用的时候自己按下显示器开关，到现在也没再遇到过。

**Reply by szh-bash** [reply]
> 应该是系统本身有问题，浏览器中的js并没有直接访问系统的能力，你完全可以排除卡住是因为浏览器的xx插件所导致的。
> 
> 我也遇到过类似的现象。我用笔记本，那段时间外接了一台显示器，设置了屏幕超时。当系统自动关闭屏幕后，我再尝试唤醒就会时不时那样，而不外接显示器只用内置显示器且有屏幕超时的时候从没出现过这个现象。索性我就关了屏幕超时，让屏幕常亮了，不用的时候自己按下显示器开关，到现在也没再遇到过。

你提醒我了唤醒后卡死我好像几年前也遇到过，机器卡死时状态和现在几乎一样，貌似通过重装系统解决了（太过久远，没做记录忘记当时具体怎么解决的，折腾了很久都没解决应该是重装才好的。。。）

现在有一个可能的因素：2.26 当天收到京东换新的 990Pro，目前的系统通过 Samsung Magician 先后由 970P->990P(old)->970P->990P(new) 迁移系统，每个阶段分别用了3年-4月-14D-30D 而**在前三个阶段以及第四个阶段的前 14 天（2.26->3.9）均未出现卡屏死机**，第四阶段应该也没有做过什么系统设置、更新等，不太明确是否有 ssd 物理故障还是 OS 故障。<del>实在是不愿意重装 OS</del>

**Comment by szh-bash** [comment, marked as answer]
重新迁移 970p 系统到 990p，再没出现卡屏死机了。。。重新升级了各种软件、OS，也再没卡屏死机过，所以还是之前 win11 系统突然自己爆炸了。。。神奇

**Reply by learnerLj** [reply]
你好，我看到了相关教程：https://rqdmap.top/posts/windows-chrome/
可能是由于这个原因，我也遇到了类似的问题。

**Reply by szh-bash** [reply]
> 你好，我看到了相关教程：https://rqdmap.top/posts/windows-chrome/ 可能是由于这个原因，我也遇到了类似的问题。

谢谢你，但我是在系统登录界面都有一定机率死机，可能和你提供的教程不太相似。
今年找维修点返修后定位是主板的cpu供电故障，修理后确实有极大好转，连续使用大半月才会死机了。不过现在只要同时打开两台VMware然后在虚拟机内启动GPU压力程序，秒死机，可能硬件方面还是有其他故障
